src/types.rs

```diff
@@ -227,3 +230,56 @@ impl SignatureVerification {
         self.inner.trusted()
     }
 }
+
+/// The result of a call to {@link OlmMachine.importExportedRoomKeys} or
+/// {@link OlmMachine.importBackedUpRoomKeys}.
+#[derive(Clone, Debug)]
+#[wasm_bindgen]
+pub struct RoomKeyImportResult {
+    /// The number of room keys that were imported.
```

Comment:
Maybe worth commenting how it might differ from `total_count`? (I've read that elsewhere in this PR, but since it's in a different file...)

Reply:
I don't really know how it might differ, if I'm honest. I could hypothesise that it would exclude keys we already know about, but investigating this seems beyond the scope of this PR.

(This is just a copy of the documentation from https://github.com/matrix-org/matrix-rust-sdk/blob/main/crates/matrix-sdk-crypto/src/lib.rs#L52.)